src/libostree/libostree.sym

```diff
@@ -373,12 +373,10 @@ global:
  *                         NOTE NOTE NOTE
  */
 
-/* Remove comment when first new symbol is added
-LIBOSTREE_2016.XX {
+LIBOSTREE_2017.3 {
```

Comment:
I was a little hesitant about this, but it seems to work. My thinking regarding leaving it as `2017.3` rather than adding it in a `2016.15` section is that it’s the same as upstream — so when we upgrade OSTree, the symbol definition won’t magically switch from the latter to the former.

I admit I have not had a chance to re-read about this style of symbol versioning. I will do that before considering this task closed.